src/conversations/Conversation.ts

```diff
@@ -247,21 +247,31 @@ export class ConversationV1<ContentTypes>
     }
     const payload = await this.client.encodeContent(content, options)
     const msg = await this.createMessage(payload, recipient, options?.timestamp)
+    const msgBytes = msg.toBytes()
 
     const env: messageApi.Envelope = {
       contentTopic: this.topic,
-      message: msg.toBytes(),
+      message: msgBytes,
       timestampNs: toNanoString(msg.sent),
     }
 
     return new PreparedMessage(env, async () => {
       await this.client.publishEnvelopes(
         topics.map((topic) => ({
           contentTopic: topic,
-          message: msg.toBytes(),
+          message: msgBytes,
           timestamp: msg.sent,
         }))
       )
+
+      return DecodedMessage.fromV1Message(
+        msg,
+        content,
+        options?.contentType || ContentTypeText,
+        payload,
+        topics[0],
```

Comment:
Think we want it to always be `this.topic`. For a V1 message, `topics[0]` will be the intro topic

Reply:
ahh OK, i originally had it like that, but then saw what the `send` method is doing. will update!

Reply:
Oof. We should change which topic gets returned from send too, then.